packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -112,8 +113,27 @@ public void onError(String code, String error) {
       } catch (Exception e) {
         result.error("no_biometrics_available", e.getMessage(), null);
       }
+    } else if (call.method.equals(("stopAuthentication"))) {
+      stopAuthentication(result);
     } else {
       result.notImplemented();
     }
   }
+
+  /*
+   Stops the authentication if in progress.
+  */
+  private void stopAuthentication(Result result) {
+    try {
+      if (authenticationHelper != null && authInProgress.get()) {
+        authenticationHelper.stopAuthentication();
```

Comment:
Doing this means `.authenticate` method will never return? Will one of the `AuthCompletionHandler` methods be called when you stop authentication?

Reply:
@mehmetf 
1. The ```.authenticate``` will return ``` false``` when we stop authentication.
2. Yes, the ```onFailure``` method will be called.